package.json

```diff
@@ -127,6 +127,7 @@
     "vega-datasets": "^2.7.0",
     "vega-embed": "^6.22.1",
     "vega-tooltip": "^0.32.0",
+    "vega-typings": "^0.24.1",
```

Comment:
Since we already depend on Vega, I don’t think we need this. It just makes updates more annoying.